nominations/views.py

```diff
@@ -507,7 +507,10 @@ class ApplicationPDFView(PDFTemplateView):
 
     def get_context_data(self, **kwargs):
         app_id = self.request.GET.get('id')
-        app = get_object_or_404(Application, pk=app_id)
+        app = get_object_or_404(
+            Application.objects.select_related(),
```

Comment:
Nice.

Reply:
I find this a little confusing but glad it looks right to you.